po/pl.po

```diff
@@ -800,7 +830,7 @@ msgid "Device: %s (%d ms)"
 msgstr "Urzadzenie: %s (%d ms)"
 
 msgid "Direct Sound:"
-msgstr "Direct Sound:"
+msgstr "Bezpośredni dźwięk:"
```

Comment:
Should this one be translated at all? Isn't this a proper noun for Microsoft's sound engine?

Reply:
No, it isn't. It is a lable on the "Reverb tab" that means whether the original sound should be mixed to the reverb.

Reply:
Oh, if that's the context then it's valid by all means.